### PR TITLE
style(line/gestion des appels) convert switch to checkbox

### PR DESCRIPTION
**Pull request by GenghisJean**

### CODE REVIEW

File: client/app/telecom/telephony/line/calls/simultaneousLines/telecom-telephony-line-calls-simultaneousLines.html [hunk 1]
```diff
@@ -35,17 +35,19 @@ <h2 data-translate="loading"></h2>
 
                     <div class="row my-4">
                         <div class="col-xs-12 col-md-6">
-                            <label for="simultaneousLines"
-                                data-translate="telephony_line_actions_line_calls_simultaneous_line_label"></label>
-                        </div>
-                        <div class="col-xs-12 col-md-6">
-                             <input-number-spinner data-ng-model="LineSimultaneousLinesCtrl.options.simultaneousLines"
+                            <label class="d-block"
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/calls/simultaneousLines/telecom-telephony-line-calls-simultaneousLines.html]: -`d-block` => `control-label`

File: client/app/telecom/telephony/line/calls/lockOutCall/telecom-telephony-line-calls-lockOutCall.html [hunk 2]
```diff
@@ -30,20 +30,24 @@ <h2 data-translate="loading"></h2>
...
-                                 type="checkbox"
-                                 class="oui-switch oui-switch__checkbox"
                                  data-ng-disabled="LineLockOutCallCtrl.loading.save"
-                                 data-ng-model="LineLockOutCallCtrl.options.lockOutCall"/>
-                          <div class="oui-switch__slider"></div>
-                          <i class="oui-icon oui-icon-success" aria-hidden="true"></i>
-                          <i class="oui-icon oui-icon-error" aria-hidden="true"></i>
-                      </label>
+                                 data-ng-model="LineLockOutCallCtrl.options.lockOutCall">
+                          <label class="oui-checkbox__label-container" for="lockOutCall">
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/calls/lockOutCall/telecom-telephony-line-calls-lockOutCall.html]: Indentation.

File: client/app/telecom/telephony/line/calls/callWaiting/telecom-telephony-line-calls-callWaiting.html [hunk 3]
```diff
@@ -66,20 +64,25 @@ <h2 data-translate="loading"></h2>
...
-                                <i class="oui-icon oui-icon-success" aria-hidden="true"></i>
-                                <i class="oui-icon oui-icon-error" aria-hidden="true"></i>
-                            </label>
+                                       data-ng-model="LineCallWaitingCtrl.options.intercomSwitch">
+                                <label class="oui-checkbox__label-container" for="intercomSwitch">
+                                    <span class="oui-checkbox__label"
+                                          data-translate="telephony_line_actions_line_calls_call_waiting_intercom_checkbox_label">
+                                    </span>
+                                    <span class="oui-checkbox__icon">
+                                      <i class="oui-icon oui-icon-checkbox-unchecked" aria-hidden="true"></i>
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/calls/callWaiting/telecom-telephony-line-calls-callWaiting.html]: Indentation.

File: client/app/telecom/telephony/line/calls/callWaiting/telecom-telephony-line-calls-callWaiting.html [hunk 4]
```diff
@@ -49,9 +49,7 @@ <h2 data-translate="loading"></h2>
                                 <i class="oui-icon oui-icon-success" aria-hidden="true"></i>
                                 <i class="oui-icon oui-icon-error" aria-hidden="true"></i>
                             </label>
-                        </div>
-                        <!-- /.widget-presentation -->
-
+                        </div><!-- /.widget-presentation -->
```
Comment (antleblanc) [inline client/app/telecom/telephony/line/calls/callWaiting/telecom-telephony-line-calls-callWaiting.html]: Any reason? :smile:

Comment (GenghisJean) [reply client/app/telecom/telephony/line/calls/callWaiting/telecom-telephony-line-calls-callWaiting.html]: be uniform to most those type of comment

**Pull request merged**